README.rst

```diff
@@ -139,9 +113,9 @@ reproducibility, please cite the version you used as indexed by
 `Zenodo <https://doi.org/10.5281/zenodo.1003157>`_.
 Or cite the generic concept as::
 
-    @misc{pygsp,
-      title = {PyGSP: Graph Signal Processing in Python},
+    @misc{pygsp2,
+      title = {PyGSP2: Graph Signal Processing in Python},
       author = {Defferrard, Micha\"el and Martin, Lionel and Pena, Rodrigo and Perraudin, Nathana\"el},
       doi = {10.5281/zenodo.1003157},
-      url = {https://github.com/epfl-lts2/pygsp/},
+      url = {https://github.com/gsp-eeg/pygsp2/},
     }
```

Comment:
Reference to the original project. We should not change this reference to the original PyGSP.

Reply:
solved